src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -80,12 +81,35 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
-    float sentimentScore = getSentimentScore(text);
 
-    Message message = new Message(user, text, sentimentScore);
-    datastore.storeMessage(message);
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
 
+    String regex = "(https?://\\S+\\.(png|jpg|gif))";
```

Comment:
Maybe add some comments to these regular expressions explaining what they're used for?

Reply:
Ok!! I'll add some comments.